graphql_client/__init__.py

```diff
@@ -45,11 +45,13 @@ def _conn_init(self, headers=None):
         self._conn.send(json.dumps(payload))
         self._conn.recv()
 
+
```

Comment:
remove this extra newline?

Reply:
@ecthiender I have removed the extra line.

Reply:
https://github.com/ecthiender/py-graphql-client/pull/3/commits/e486ccb616fb3286247dce50f14486a784e7444e